website/versioned_docs/version-0.62/supported-community-modules.md

```diff
@@ -0,0 +1,50 @@
+---
+id: version-0.62-supported-community-modules
+title: Supported Community Modules
+original_id: supported-community-modules
+---
+
+## ReactNative.Directory
+
+The [React Native Directory](https://reactnative.directory/) lists libraries and native modules that are available across all of the React Native platforms including iOS, Android, Windows, macOS, and more. 
+
+To view which modules are available for a specific platform, you can use the Filters function on the website, or visit these pre-filtered URLs:
+
+- [Modules that support Windows](https://reactnative.directory/?windows=true)
+- [Modules that support macOS](https://reactnative.directory/?macos=true)
+
+## Microsoft-supported Modules
+
+The React Native team at Microsoft has worked together with the community to add Windows and macOS support to several community modules. The modules that Microsoft has added support for are listed below. 
+
+The following modules have been ejected as part of the React Native [Lean Core](https://github.com/facebook/react-native/issues/23313) effort:
+
+| Name | Version Supported (Windows) | Version Supported (macOS) |
+|:-|:-|:-|
+| <ins>[async-storage](https://github.com/react-native-community/async-storage)</ins> | 0.61 | 0.61 |
+| <ins>[datetimepicker](https://github.com/react-native-community/datetimepicker)</ins> | 0.62 | ? |
+| <ins>[react-native-checkbox](https://github.com/react-native-community/react-native-checkbox)</ins> | 0.62 | ? |
+| <ins>[react-native-netinfo](https://www.github.com/react-native-community/react-native-netinfo)</ins> | 0.61 | 0.61 |
+| <ins>[react-native-picker](https://github.com/react-native-community/react-native-picker)</ins> | 0.61 | 0.61 |
```

Comment:
There is no picker in the community module.  An impl exists in react-native-macos but someone needs to port it to the community repo (https://github.com/microsoft/react-native-macos/issues/395)